featurebyte/routes/use_case/controller.py

```diff
@@ -50,25 +45,6 @@ async def create_use_case(self, data: UseCaseCreate) -> UseCaseModel:
         """
         return await self.service.create_use_case(data)
 
-    async def get_use_case(self, use_case_id: ObjectId) -> UseCaseRead:
```

Comment:
why are these being removed?

Reply:
previously it needs to return the Target/Context objects as part of the attributes in the UseCase API object. But now since we just use target_id/context_id as the attributes, the raw get/list routes can already satisfy the attributes, so no need these functions any more.